src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -2426,13 +2376,12 @@ namespace VariableSpeedCoils {
                     ErrorsFound = true;
                 } else {
                     // Verify Curve Object, only legal type is Quadratic
-                    ErrorsFound |= CurveManager::CheckCurveDims(
-                        VarSpeedCoil(DXCoilNum).MSEIRWaterFFlow(I),   // Curve index
-                        {1},                            // Valid dimensions
-                        RoutineName,                    // Routine name
-                        CurrentModuleObject,            // Object Type
-                        VarSpeedCoil(DXCoilNum).Name,   // Object Name
-                        cAlphaFields(AlfaFieldIncre));  // Field Name
+                    ErrorsFound |= CurveManager::CheckCurveDims(VarSpeedCoil(DXCoilNum).MSEIRWaterFFlow(I), // Curve index
+                                                                {1},                                        // Valid dimensions
+                                                                RoutineName,                                // Routine name
+                                                                CurrentModuleObject,                        // Object Type
+                                                                VarSpeedCoil(DXCoilNum).Name,               // Object Name
+                                                                cAlphaFields(AlfaFieldIncre));              // Field Name
```

Comment:
@Myoldmopar So, seems like Visual Studio is interpreting the clang format differently from linux?